spec/models/intake_spec.rb

```diff
@@ -248,58 +235,6 @@
     end
   end
 
-  describe "#users" do
-    it "should have many users" do
-      relation = Intake.reflect_on_association(:users).macro
-      expect(relation).to eq :has_many
-    end
-  end
-
-  describe "#primary_user" do
-    let(:intake) { create :intake }
-
-    context "with no user" do
-      it "returns nil" do
-        expect(intake.primary_user).to be_nil
-      end
-    end
-
-    context "with a couple filing jointly" do
-      let!(:primary) { create :user, intake: intake }
-      let!(:spouse) { create :spouse_user, intake: intake }
-
-      it "returns the first non-spouse user" do
-        expect(intake.primary_user).to eq primary
-      end
-    end
-
-    context "with one user" do
-      let!(:primary) { create :user, intake: intake }
-
-      it "returns that one user" do
-        expect(intake.primary_user).to eq primary
-      end
-    end
-  end
```

Comment:
I don't think we should remove these tests for the associations yet. We may need to use these to debug issues that arise after we make the change.